src/Blocks/Input.purs

```diff
@@ -1,28 +1,72 @@
-module CN.UI.Block.Input (input) where
+module CN.UI.Block.Input where
 
 import Prelude
 
-import DOM.HTML.Indexed (HTMLinput)
+import DOM.HTML.Indexed (HTMLinput, HTMLlabel)
+import Data.Maybe (Maybe, maybe)
 import Halogen.HTML as HH
 import Halogen.HTML.Properties as HP
 
-inputClasses :: Array HH.ClassName
-inputClasses = HH.ClassName <$>
+inputOuterClasses :: Array HH.ClassName
+inputOuterClasses = HH.ClassName <$>
   [ "bg-white"
-  , "cc-blue-88"
+  , "font-light"
+  -- , "border-b"
+  -- , "border-grey-lighter"
+  , "rounded-sm"
+  , "w-full"
+  , "flex"
+  ]
+
+inputInnerClasses :: Array HH.ClassName
+inputInnerClasses = HH.ClassName <$>
+  [ "cc-blue-88"
   , "disabled:bg-grey-light"
   , "focus:no-outline"
-  , "font-light"
   , "px-4"
-  , "py-2"
+  , "py-3"
   , "rounded-sm"
-  , "text-sm"
-  , "w-full"
+  , "flex-auto"
+  ]
+
+inputClasses :: Array HH.ClassName
+inputClasses = inputOuterClasses <> inputInnerClasses
+
+inputLeftClasses :: Array HH.ClassName
+inputLeftClasses = HH.ClassName <$>
+  [ "border-r"
+  , "ml-r"
+  , "px-3"
+  , "cursor-pointer"
+  , "self-center"
+  ]
+
+inputRightClasses :: Array HH.ClassName
+inputRightClasses = HH.ClassName <$>
+  [ "border-l"
+  , "ml-1"
+  , "px-3"
+  , "cursor-pointer"
+  , "self-center"
   ]
 
 input
   :: ∀ p i
-  . Array (HH.IProp HTMLinput i)
+   . Array (HH.IProp HTMLinput i)
   -> HH.HTML p i
 input iprops =
   HH.input (iprops <> [ HP.classes inputClasses ])
+
+-- inputGroup
```

Comment:
do we need these commented out blocks? i'd rather not have commented out code get onto master